docs/tutorials/go-built-in.md

```diff
@@ -585,17 +578,37 @@ We are going to use [Go modules](https://github.com/golang/go/wiki/Modules) for
 dependency management.
 
 ```bash
+export GO111MODULE=on
 go mod init github.com/me/example
-go build
 ```
 
-This should build the binary.
+This should create a `go.mod` file. The current tutorial only works with
+tendermint > v0.34, so let's make sure we're using the latest version:
+
+```
+module github.com/me/example
+
+go 1.15
+
+require (
+	github.com/dgraph-io/badger v1.6.2
+	github.com/tendermint/tendermint v0.34.0-rc4
+)
+```
+
+Now we can build the binary:
+
+```bash
+go build
```

Comment:
Does this not have to point to something or use ./...?

Reply:
no. current dir is used by default